### DISCUSSION TITLE
How can I change the highlight group in vim-illuminate?

### DISCUSSION THREAD

**Discussion by neuromaancer** [LazyVim/LazyVim#2203] [Q&A] [answered]
This is strange. I have tried what the [vim-illuminate](https://github.com/RRethy/vim-illuminate/tree/master) to change the `IlluminatedWordText` group with put:
```lua
﻿﻿﻿config = function()
            vim.cmd([[hi illuminatedWordText cterm=underline gui=undercurl]])
        end,
```
in my `colorscheme.lua` (catppuccin), but it didn't change anything.

**Comment by dpetka2001** [comment, marked as answer]
It probably gets overwritten by a plugin (maybe illuminate itself because the colorscheme loads first)? Best to try with an autocmd in your `/lua/config/autocmds.lua`
```lua
vim.api.nvim_create_autocmd("BufReadPost", {
  group = vim.api.nvim_create_augroup("Change_Illuminate", { clear = true }),
  callback = function()
    vim.cmd([[hi IlluminatedWordText cterm=underline gui=undercurl]])
    vim.cmd([[hi IlluminatedWordRead cterm=underline gui=undercurl]])
    vim.cmd([[hi IlluminatedWordWrite cterm=underline gui=undercurl]])
  end,
  desc = "Change Illuminate Highlights",
})
```
Test to see if it works how you want it to.

**Reply by neuromaancer** [reply]
Yes, you are right! It works. I think it is illuminate itself.
